src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.scss

```diff
@@ -0,0 +1,7 @@
+@import "scss/constants";
+
+.profile-avatar-container {
+  width: 50px;
+  height: 50px;
+  margin-right: 10px;
+}
```

Comment:
since we have `--large`, can we get rid of this?

Reply:
shrieked to 40px

Reply:
https://github.com/sparkletown/sparkle/pull/1384/commits/5bcc570daf621979becff60da99f34b765b09c03